_ja/overviews/macros/usecases.md

```diff
@@ -20,7 +20,7 @@ Scala の商用ユーザと研究ユーザの両方がマクロを利用して
 ここ EPFL においても我々はマクロを活用して研究を行っている。Lightbend 社もマクロを数々のプロジェクトに採用している。
 マクロはコミュニティー内でも人気があり、既にいくつかの興味深い応用が現れている。
 
-最近行われた講演の ["What Are Macros Good For?"](https://scalamacros.org/paperstalks/2014-02-04-WhatAreMacrosGoodFor.pdf) では Scala 2.10 ユーザのマクロの利用方法を説明し、システム化した。講演の大筋はマクロはコード生成、静的な検査、および DSL に有効であるということで、これを研究や産業からの例を交えながら説明した。
+最近行われた講演の ["What Are Macros Good For?"](hxxps://scalamacros.org/paperstalks/2014-02-04-WhatAreMacrosGoodFor.pdf) では Scala 2.10 ユーザのマクロの利用方法を説明し、システム化した。講演の大筋はマクロはコード生成、静的な検査、および DSL に有効であるということで、これを研究や産業からの例を交えながら説明した。
```

Comment:
`hxxps` seems to be working at intended too

Reply:
Matt found a better URL and we'll use that one.